docs/_components/form-fields.md

```diff
@@ -159,6 +159,16 @@ Three styles of radio buttons are provided.
       <input id="a8c1" class="usa-radio__input usa-radio__input--tile" type="radio" name="a8c1">
       <label for="a8c1" class="usa-radio__label">Option - Unselected<span class="usa-radio__label-description">Support text</span></label>
     </li>
+    <li>
+      <input id="abc123" class="usa-radio__input usa-radio__input--tile" type="radio" name="abc123">
+      <label for="abc123" class="usa-radio__label usa-radio__label-illustrated">
+        <img class="usa-radio__image" src="{{ site.baseurl}}/assets/img/fingerprint-icon.svg">
+        <div class="usa-radio__label-text">
+          <span>Option - Illustration</span>
```

Comment:
It would be nice if we could avoid this extra span. Is it needed?

Reply:
It was not needed, so I removed it.